src/icons.scss

```diff
@@ -687,4 +687,27 @@
     @extend .fa;
     @extend .fa-refresh;
   }
+
+  // Misc
+
+  &-search {
+    @extend .fa;
+    @extend .fa-fw;
+    @extend .fa-search;
+  }
+
+  // Alias
```

Comment:
I agree, some very "general" icons like "search" or "start", should not be created in a specific context like "menu" or "vm". I was waiting for another opinion on the matter.

Do you wish some changes in this PR, or a specific PR to rework the icon set ?